dmriprep/cli/parser.py

```diff
@@ -216,6 +216,23 @@ def _bids_filter(value):
 https://www.nipreps.org/dmriprep/en/%s/spaces.html"""
         % (currentv.base_version if is_release else "latest"),
     )
+    g_conf.add_argument(
+        "--bold2t1w-init",
+        action="store",
+        default="register",
+        choices=["register", "header"],
+        help='Either "register" (the default) to initialize volumes at center or "header"'
+        " to use the header information when coregistering BOLD to T1w images.",
+    )
+    g_conf.add_argument(
+        "--bold2t1w-dof",
+        action="store",
+        default=6,
+        choices=[6, 9, 12],
+        type=int,
+        help="Degrees of freedom when registering BOLD to T1w images. "
+        "6 degrees (rotation and translation) are used by default.",
+    )
```

Comment:
Let's not allow changing this. IMHO this should not exist in *fMRIPrep* either, we can call it legacy code that has not been cleaned up for historical reasons (and lack of bandwidth)
```suggestion
```

Reply:
I was wondering about this, that clears it up.